Certification-Standard/introduction.md

```diff
@@ -0,0 +1,239 @@
+
+**Attack Surface**: Set of logical interfaces to a target, consisting of points through which access to the target's assets or control.
+*RSoulat* I removed "physical" and changed a bit the definition from the CC definition to better fit (I believe) what we're doing.
+
+**Decentralized Application**: Set of software running both on the blockchain, smart contracts, and on classical servers, off-chain code.
+
+**Developer**: Organization responsible for the development of the Target of Evaluation
+
+**Entity**: Identifiable item that is described by a set of properties. For example: subjects, users, objects, information sessions, resources
+*RSoulat* We could include Ada, Tokens, NFTs, Wallets, etc in the Entity definition?
+
+**Evaluation Level**: Well formed package of security assurance requirements representing a point on the predefined assurance scale.
+
+**Evaluation authority**: Body operating an evaluation scheme
+*RSoulat* Could be Auditors, or any "certificate provider", the difference with the CC is that the evaluation authority is **not** the certificate issuer
+
+**Exploitable vulnerability**: Weakness in the Target of Evaluation (TOE) that can be used to violate the Security Functional Requirement (SFRs) in the Operation Environment (OE) for the TOE
+
+**Object**: Entity in the Target of Evaluation that contains or receives information and upon which subjects perform operations.
+*RSoulat* Maybe we could also augment this definition to include UTxOs? 
+
+**Operation Environment**: Enfironment in which the Target of Evaluation (TOE) is operated, consisting of everything that is outside the TOE boundary
+*RSoulat* For example, the ledger, the other contracts, the off-chain code if the TOE is only the on-chain component
+
+**Potential vulnerability**: Suspected, but not confirmed, weakness
+
+**Protection Profile (PP)**: Implementation-independant statement of the security needs for a Target of Evaluation (TOE) type
+
+**Residual Vulnerability**: Weakness that cannot be exploited in the Operational Environment for the Target of Evaluation but that can be used to violate a Security Functional Requirement (SFR) by an attacker with greater Attack Potential than is anticipated in the operational environment for the TOE.
+
+**Security Assurance Requirements**: Security requirements that refers to the conditions and processes for the development and delivery of the Target of Evaluation (TOE) and the actions required of evaluators with respect to evidence produced from these conditions and processes.
+
+**Security Functional Requirements**: Security requirements, which contributes to fulfil the Target of Evaluation (TOE) Security Problem Definition (SPD) as defined in a specific Security Target (ST) or in a Protection Profile (PP).
+
+**Security Objective**: Statement of an intent to counter identified threats and/or identified organization security policies and/or assumptions.
```

Comment:
There is anecdotal evidence that there is a correlation between good development practices and better security properties of DApps.

Reply:
Absolutely. But I don't think we can include that as a "security objective" it's actually part of the ADV activities in CC.